components/inputs/input-text.js

```diff
@@ -376,8 +376,8 @@ class InputText extends FormElementMixin(SkeletonMixin(RtlMixin(LitElement))) {
 						tabindex="${ifDefined(this.tabindex)}"
 						title="${ifDefined(this.title)}"
 						type="${this._getType()}">
-					<div class="d2l-input-inside-before">${this.dir === 'rtl' ? unit : ''}<slot name="${firstSlotName}" @slotchange="${this._handleSlotChange}"></slot></div>
-					<div class="d2l-input-inside-after">${this.dir !== 'rtl' ? unit : ''}<slot name="${lastSlotName}" @slotchange="${this._handleSlotChange}"></slot></div>
+					<div class="d2l-input-inside-before" @keypress="${this._suppressEvent}">${this.dir === 'rtl' ? unit : ''}<slot name="${firstSlotName}" @slotchange="${this._handleSlotChange}"></slot></div>
+					<div class="d2l-input-inside-after" @keypress="${this._suppressEvent}">${this.dir !== 'rtl' ? unit : ''}<slot name="${lastSlotName}" @slotchange="${this._handleSlotChange}"></slot></div>
```

Comment:
One bug this change exposed is that because search input was listening for `keypress` on the input to detect when ENTER was pressed, when you focus on the search/clear buttons in the slots and hit ENTER it was interpreting that the same way. So we want to prevent these events from bubbling up.

Reply:
Another way this could be handled would be to have `d2l-input-search`'s `_handleInputKeyPress` check the target and bail if it were any of its buttons.  I could see reasons for going either way.

Reply:
I considered that, but figured this fix belonged more in `d2l-input-text` since consumers would be expecting `@keypress` and `@input` events to be originating from the text input, not stuff in the slots.